packages/dds/tree/src/test/forestTestSuite.ts

```diff
@@ -118,19 +107,18 @@ export function testForest(config: ForestTestConfiguration): void {
 			];
 			for (const [name, data] of testCases) {
 				it(name, () => {
-					const schema = new TreeStoredSchemaRepository();
-					const forest = factory(schema);
+					const schemaFactory = new SchemaFactory("forest test suite");
 
-					const rootFieldSchema = FlexFieldSchema.create(FieldKinds.optional, jsonRoot).stored;
-					schema.apply({
-						nodeSchema: new Map(
-							mapIterable(jsonSchema.nodeSchema.entries(), ([k, v]) => [k, v.stored]),
-						),
-						rootFieldSchema,
-					});
+					const rootSchema = schemaFactory.optional([JsonArray]);
+					const schema = new TreeStoredSchemaRepository(
+						intoStoredSchema(toFlexSchema(rootSchema)),
+					);
+
+					const forest = factory(schema);
 
+					// TODO: fix this if we really want it and remove otherwise
 					// Check schema is actually valid. If we forgot to add some required types this would fail.
-					assert(!isNeverField(defaultSchemaPolicy, schema, rootFieldSchema));
+					// assert(!isNeverField(defaultSchemaPolicy, schema, rootFieldSchema));
```

Comment:
This assert really only had value when our APIs for making schema were horrible and it was supper easy to get things very wrong, like accidently making a schema which was impossible to satisfy (which is what this detected). Removing ti should be fine as we should be quite confident that JsonArray isn't invalid in that particular way.